js/main.js

```diff
@@ -0,0 +1,127 @@
+$(document).ready(function () {
+
+    $("#loginForm").submit(function (event) {
+
+        var email = $("#email").val();
+        var password = $("#pwd").val();
+
+        $.ajax({
```

Comment:
$.ajax este un api deprecated in JQuery, foloseste $.post si $.get pentru standard request-types. 

$.ajax s-a transformat intr-un low-level api, folosit doar pentru request-urile de tip "PUT", "DELETE", etc.